mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -751,6 +751,14 @@ func resourceComputeInstance() *schema.Resource {
 							AtLeastOneOf: []string{"advanced_machine_features.0.enable_nested_virtualization","advanced_machine_features.0.threads_per_core"},
 							Description: `The number of threads per physical core. To disable simultaneous multithreading (SMT) set this to 1. If unset, the maximum number of threads supported per core by the underlying processor is assumed.`,
 						},
+<% unless version == 'ga' -%>
+						"visible_core_count": {
+							Type:        schema.TypeInt,
+							Optional:    true,
+							AtLeastOneOf: []string{"advanced_machine_features.0.enable_nested_virtualization","advanced_machine_features.0.visible_core_count"},
```

Comment:
Can you add the visible_core_count to the AtLeastOneOf of the above fields as well? Just for consistency's sake

Reply:
Fixed.